engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/source/BaseKafkaAvroSourceFactory.scala

```diff
@@ -23,19 +25,22 @@ abstract class BaseKafkaAvroSourceFactory[T: ClassTag](processObjectDependencies
   // We currently not using processMetaData and nodeId but it is here in case if someone want to use e.g. some additional fields
   // in their own concrete implementation
   def createSource(preparedTopic: PreparedKafkaTopic,
+                   version: Option[Int],
                    kafkaConfig: KafkaConfig,
-                   kafkaAvroSchemaProvider: KafkaAvroSchemaProvider[T],
+                   schemaRegistryProvider: SchemaRegistryProvider[T],
```

Comment:
The same as in Sink. There sould be only: deserializationSchemaFactory and recordFormatter.